ydb/src/client_table_test_integration.rs

```diff
@@ -1,622 +1,733 @@
-        let mut values = Vec::with_capacity(target_batch_size);
-        for _ in 0..target_batch_size {
-            last_item_value += 1;
-            expected_sum += last_item_value;
-            values.push(last_item_value);
-        }
-        insert_values(&client, values).await?;
-    }
-    let expected_item_count = last_item_value;
-
-    let mut expected_id: i64 = 1;
-    let query = Query::new("SELECT * FROM stream_query ORDER BY id".to_string());
-    let mut res = session.execute_scan_query(query).await?;
-    let mut sum: i64 = 0;
-    let mut item_count = 0;
-    let mut result_set_count = 0;
-    while let Some(result_set) = res.next().await? {
-        result_set_count += 1;
-
-        for mut row in result_set.into_iter() {
-            item_count += 1;
-            match row.remove_field_by_name("id")? {
-                Value::Optional(boxed_id) => match boxed_id.value.unwrap() {
-                    Value::Int64(id) => {
-                        assert_eq!(id, expected_id);
-                        sum += id
-                    }
-                    id => panic!("unexpected ydb boxed_id type: {:?}", id),
-                },
-                id => panic!("unexpected ydb id type: {:?}", id),
-            };
-
-            match row.remove_field_by_name("val")? {
-                Value::Optional(boxed_val) => match boxed_val.value.unwrap() {
-                    Value::String(content) => {
-                        assert_eq!(gen_value_by_id(expected_id), Vec::<u8>::from(content))
-                    }
-                    val => panic!("unexpected ydb id type: {:?}", val),
-                },
-                val => panic!("unexpected ydb boxed_id type: {:?}", val),
-            };
-
-            expected_id += 1;
-        }
-    }
-
-    assert_eq!(expected_item_count, item_count);
-    assert_eq!(expected_sum, sum);
-
-    // TODO: need improove for non flap in tests for will strong more then 1
-    assert!(result_set_count > 1); // ensure get multiply results
-    Ok(())
-}
+use std::collections::HashMap;
+use std::iter::FromIterator;
+use std::sync::{Arc, Mutex};
+use std::time;
+use std::time::UNIX_EPOCH;
+
+use tonic::{Code, Status};
+use tracing::trace;
+use tracing_test::traced_test;
+
+use crate::client_table::RetryOptions;
+use crate::errors::{YdbError, YdbOrCustomerError, YdbResult};
+use crate::query::Query;
+use crate::test_integration_helper::create_client;
+use crate::transaction::Mode;
+use crate::transaction::Transaction;
+use crate::types::{Value, ValueList, ValueStruct};
+use crate::{ydb_params, Bytes, TableClient};
+
+#[tokio::test]
+#[traced_test]
+#[ignore] // need YDB access
+async fn create_session() -> YdbResult<()> {
+    let res = create_client()
+        .await?
+        .table_client()
+        .create_session()
+        .await?;
+    trace!("session: {:?}", res);
+    Ok(())
+}
+
+#[tokio::test]
+#[traced_test]
+#[ignore] // need YDB access
+async fn execute_data_query() -> YdbResult<()> {

```

Comment:
Is it change new line delimiter?

All file shown as replaced

need to check the change for possible to view changes only